README.md

```diff
@@ -235,7 +235,12 @@ label the issue as **in progress**. For more details, check out
 
 You're encouraged to ask questions on how to best implement or debug
 your changes -- the Zulip maintainers are excited to answer questions
-to help you stay unblocked and working efficiently.
+to help you stay unblocked and working efficiently. It's great to ask
+questions in comments on GitHub issues and pull requests, or [on
+chat.zulip.org](https://chat.zulip.org/). We'll direct longer
```

Comment:
please make the link go to `chat-zulip-org.html` (the documentation page about it) so that people get context

otherwise this looks good!